pkg/limayaml/defaults.go

```diff
@@ -106,7 +106,11 @@ func FillDefault(y, d, o *LimaYAML, filePath string) {
 	}
 	for arch := range cpuType {
 		if IsNativeArch(arch) && IsAccelOS() {
-			cpuType[arch] = "host"
+			if HasHostCPU() {
+				cpuType[arch] = "host"
+			} else {
+				cpuType[arch] = "max"
+			}
```

Comment:
Maybe this should be a function, as the logic is duplicated in the test?

Reply:
Yeah, the flags and functions are a bit spread out between qemu and limayaml